### PR TITLE
Add usage metrics for counting entity updates

### PR DESCRIPTION
**Pull request by ktuite**
Backend portion of https://github.com/getodk/central/issues/410
Needs the frontend translation change and the central config version change and the new version of the form to be uploaded.

[odk-analytics.xlsx](https://github.com/getodk/central-backend/files/11566705/odk-analytics.xlsx)

#### What has been done to verify that this works as intended?

Integration Test +

Tried sending local analytics to test server.

image

#### Why is this the best possible solution? Were any other approaches considered?

NA

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

No

#### Does this change require updates to the API documentation? If so, please update docs/api.md as part of this PR.

No

#### Before submitting this PR, please make sure you have:

- [ ] run `make test-full` and confirmed all checks still pass OR confirm CircleCI build passes
- [ ] verified that any code from external sources are properly credited in comments or that everything is internally sourced

### CODE REVIEW

File: lib/data/analytics.js [hunk 1]
```diff
@@ -154,6 +154,10 @@ const metricsTemplate = {
         "num_failed_entities": {
           "recent": 0,
           "total": 0
+        },
+        "num_updated_entities": {
```
Comment (matthew-white) [inline lib/data/analytics.js:158]: I think this name has the potential to be slightly confusing. We're measuring the number of entity updates, where each entity can be counted multiple times (once for each update). However, the name of the metric seems to imply that we're measuring a subset of all entities. Put another way, it's possible for `num_updated_entities` > `num_entities`.

@sadiqkhoja, I'll let you decide whether to use the existing name or a different name like `num_entity_updates`. The name isn't shown in Frontend (text is shown instead), so it's just a question of whether we could end up getting confused internally.

Comment (sadiqkhoja) [reply lib/data/analytics.js:158]: Do we want to count "Number of Entities Updated" or "Number of Updates across all Entities"?

Comment (matthew-white) [reply lib/data/analytics.js:158]: I'm not sure actually. getodk/central#410 suggests the latter. "Number of Entities Updated" would never exceed "Number of Entities", but I think it'd be interesting if we saw a very high "Number of Updates across all Entities". It'd be nice to have some sense of how often updates happen.

File: lib/model/query/analytics.js [hunk 2]
```diff
@@ -392,6 +394,16 @@ FROM datasets ds
     WHERE a."action" = 'entity.create.error'
     GROUP BY dfd."datasetId"
   ) AS errors ON ds.id = errors."datasetId"
+  LEFT JOIN (
+    SELECT
+      COUNT (a.details -> 'entityId'::TEXT) total,
```
Comment (matthew-white) [inline lib/model/query/analytics.js:399]: Could this just be `COUNT(1)`?

**Review by matthew-white — APPROVED** [review]
I think @sadiqkhoja did the primary review. I took a quick look at the new query. I also confirmed that `num_updated_entities` has been renamed to `num_entity_updates`.

**Comment by matthew-white** [comment]
@sadiqkhoja, I see that you updated the XLSForm as well. Thanks for doing that! I've uploaded it to data.getodk.cloud.

**Pull request merged**